test/integration/switchover_test.lua

```diff
@@ -521,6 +521,57 @@ function g.test_enabling()
     t.assert_equals(g.session:get_coordinator(), box.NULL)
 end
 
+function g.test_api()
```

Comment:
Please cover case when replicaset/instance doesn't exist and instance doesn't belong to replicaset